### ISSUE TITLE
Make Tooltips Accessible by Keyboard

### ISSUE THREAD

**Issue by Scott-James-Hurley** [UserOfficeProject/user-office-project-issue-tracker#1670]
Tooltips, such as the ones found on the technique proposals page, are only accessible by mouse. Users should be able to access view them with just keyboard input and they should be read out by screen readers.
